app/src/test/java/com/xpay/kotlinutils/XpayUtilsTest.kt

```diff
@@ -88,16 +73,64 @@ class XpayUtilsTest {
         assertEquals(request.getHeader("x-api-key"), XpayUtils.apiKey)
     }
 
+
+    // check that prepare amount sets payment options successfully
+    @Test
+    fun prepareAmount_setsPaymentOptionsSuccessfully_isPassed() {
+        XpayUtils.apiKey = "3uBD5mrj.3HSCm46V7xJ5yfIkPb2gBOIUFH4Ks0Ss"
+        XpayUtils.communityId = "zogDmQW"
+        XpayUtils.variableAmountID = 18
+        XpayUtils.request = serviceRequest
+        val prepareAmountResponseBody = FileUtils.readTestResourceFile("PrepareAmountResponse.json")
+        mockWebServer.enqueue(MockResponse().setBody(FileUtils.readTestResourceFile("PrepareAmountResponse.json")))
+
+        runBlocking {
+            XpayUtils.prepareAmount(80)
+        }
+        // assertion
+        assertFalse(XpayUtils.activePaymentMethods.isEmpty());
+    }
```

Comment:
This assertion is not enough.
You need to assert that XpayUtils.activePaymentMethods is equal to MutableList<PaymentMethods> that contains payment methods based on  the response you enque. 

For example, if you enque a response that includes card, cash, and kiosk.
you need to assert that xpayutils,activePaymentMethods equal to a  MutableList<PaymentMethods> that contains card, cash , and kiosk.